src/profiler.h

```diff
@@ -68,7 +68,7 @@ class Profiler {
     Engine* _alloc_engine;
     int _event_mask;
 
-    time_t _start_time;
+    u64 _start_time;
     time_t _stop_time;
```

Comment:
`_start_time` and `_stop_time` are on the same scale. They should be measured with the same function in the same units.